nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/process-group/create-process-group/_create-process-group.component-theme.scss

```diff
@@ -26,13 +26,13 @@
     $accent-palette: map.get($color-config, 'accent');
 
     // Get hues from palette
-    $accent-palette-A400: mat.get-color-from-palette($accent-palette, 'A400');
+    $accent-palette-default: mat.get-color-from-palette($accent-palette, 'default');
 
     // Get hues from palette
```

Comment:
Comment???

Reply:
Not introduced in this PR, but worth fixing